File: 2021W29/B0neh3ad/pgr_12945.py
'''
    피보나치 수 만들기
    https://programmers.co.kr/learn/courses/30/lessons/12945
'''
def solution(n):
    a, b, c = 0, 1, 1
    for i in range(n):
        c = (a+b)%1234567
        a = b
        b = c
    return a

File: 2021W29/fienestar/pgr_12945.js
/*
    피보나치 수
    https://programmers.co.kr/learn/courses/30/lessons/12945
*/
function solution(n)
{
    let fibo = [0,-1,1]
    for(let i=0; i<=n; ++i)
        fibo[i%3] = (fibo[(i+2)%3] + fibo[(i+1)%3]) % 1234567
    return fibo[n%3]
}